server/game/gamesteps/taxationphase.js

```diff
@@ -36,6 +36,24 @@ class TaxationPhase extends Phase {
 
     roundEnded() {
         this.game.raiseEvent('onRoundEnded');
+
+        let players = this.game.getPlayers();
+        let playerStr = '';
+        let firstPlayer = true;
+
+        _.each(players, player => {
+            if(firstPlayer) {
+                firstPlayer = false;
+            } else {
+                playerStr += ', ';
+            }
+
+            playerStr += `${player.name}: ${player.getTotalPower()}`;
```

Comment:
Something like this would be more succinct:
```javascript
let playerStr = players.map(player => `${player.name}: ${player.getTotalPower()}`).join(', ')
```